test/tests/functional/pbs_highreslog.py

```diff
@@ -98,17 +97,15 @@ def test_basic(self):
         self.du.set_pbs_config(confs=a, append=True)
         _msg = 'Failed to restart server: %s' % (self.server.shortname)
         self.assertTrue(self.server.restart(), _msg)
+        print self.du.parse_pbs_config()
```

Comment:
Please use log instead of printing directly.

Reply:
Or I can just delete it.  It's a debug message I forgot to remove :)